### PR TITLE
feat: warn user for unikname TLD like creation and disclose

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

**Comment by dlecan** [comment]
https://spacelephant.tpondemand.com/entity/3178

File: src/commands/unik/disclose.ts [hunk 1]
```diff
@@ -107,20 +115,45 @@ export class UnikDiscloseCommand extends WriteCommand {
...
             } catch (e) {
                 this.debug(`disclose-explicit-values - ${e.message}`);
                 throw new Error("At least one expliciteValue does not match expected format");
             }
         }
 
+        if (hasTDLExplicitValue && unikType !== "INDIVIDUAL") {
+            const userConsent: boolean = await this.warnUserAndGetConsent(
+                `${
+                    listOfExplicitValues.length > 1 ? "At least one @uniknameID of your list" : "This @uniknameID"
```
Comment (peterjah) [inline src/commands/unik/disclose.ts:148]: why using @uniknameID and @unikname ? I don't see the point... but maybe Iam too stupid to understand the product ^^

**Review by peterjah — APPROVED** [review]

File: src/commands/unik/disclose.ts (hunk 1 above)
Comment (ftreguer) [reply src/commands/unik/disclose.ts:148]: Spec, see with PO ;)

Comment (peterjah) [reply src/commands/unik/disclose.ts:148]: je savais que t'allais dire ça !! tu donne pas ton avis ? Pr validée de mon coté.

**Pull request merged**